troubleshooting/troubleshooting-android-push-notifications-stopped-working.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Push Notifications to Android stopped working
+description: Check your Google API key if you receive the error "Error authenticating to Google Cloud Messaging"
+type: troubleshooting
+page_title:  Unsuccessful authentication to Google Cloud Messaging
+slug: troubleshooting-android-push-notifications-stopped-working
+position:
+tags: Google Cloud Messaging, GCM, Android, push notifications, FCM
+teampulseid: 
+ticketid: 1044096, 1045723
+pitsid:
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Notifications (Backend Services)</td>	
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Push Notifications, Android</td>	
+  </tr>
+</table>
+
+## Description
+
+Your Android app may have stopped receiving push notifications even they have been working for a significant amount of time. [Checking the reason for failing push notifications](http://docs.telerik.com/platform/backend-services/javascript/push-notifications/troubleshooting/push-trb-all.html#q-how-do-i-find-out-what-is-the-reason-for-not-receiving-push-notifications) in {{site.tp}} reveals the following error message. 
+
+## Error Message
+
+`Error authenticating to Google Cloud Messaging server.`
+
+## Cause
+
+The error signifies that {{site.tp}} cannot connect to Google Cloud Messaging (GCM) which is the vendor service responsible for delivering push notifications to the Android devices registered in your {{site.tp}} app. Due to recent changes in the GCM infrastructure this is the only type of key supported for server-to-server  communication. In {{site.tp}} you may have entered a Google API key of a type which is no longer supported by Google for authentication to GCM. 
+
+## Solution
+
+Verify that the Google API key entered in {{site.tp}} for your app is a **Server** API key. 
+
+Check the type of your Android API key:
+
+1. Open the [Google Developers console](https://console.developers.google.com/apis/library)
+1. From the projects list select your project - match its number to the number used to subscribe the mobile app for push notifications (as `senderId`). 
+1. On the left, click **Credentials**
+1. In the **API keys** list compare the Google API key used in {{site.tp}}
+1. Check if this key is of type **Server**. Other key types like *Android*, *Browser*, *iOS* are not supported. If the latter, you need to issue a **Server** API key and enter it in {{site.tp}}. 
+
+Issue a **Server** API key
```

Comment:
The whole Issue a Server API key area shouldn't be here. It should be in a separate KB or linked externally. If you think that its place is in this KB, move it in the Notes section!